rt_utils/rtstruct_builder.py

```diff
@@ -79,7 +79,13 @@ def validate_contour_image_in_series_data(
                 return
 
         # ReferencedSOPInstanceUID is NOT available
-        raise Exception(
-            f"Loaded RTStruct references image(s) that are not contained in input series data. "
-            + f"Problematic image has SOP Instance Id: {contour_image.ReferencedSOPInstanceUID}"
-        )
+        if warning_only:
+            warnings.warn(
+                f"Loaded RTStruct references image(s) that are not contained in input series data. "
+                + f"Problematic image has SOP Instance Id: {contour_image.ReferencedSOPInstanceUID}"
+            )
+        else:
+            raise Exception(
+                f"Loaded RTStruct references image(s) that are not contained in input series data. "
+                + f"Problematic image has SOP Instance Id: {contour_image.ReferencedSOPInstanceUID}"
```

Comment:
Lets extract the message in a common variable and reuse within the two cases

Reply:
Absolutely! Done here [41aa821](https://github.com/qurit/rt-utils/pull/64/commits/41aa8211d61f0f826fb3dcced76e0ed772269bb6)